csrc/scheduler/expr_eval_sched.h

```diff
@@ -0,0 +1,66 @@
+// clang-format off
+/*
+ * SPDX-FileCopyrightText: Copyright (c) 2023-present NVIDIA CORPORATION & AFFILIATES.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+// clang-format on
+#pragma once
+
+#include <scheduler/heuristic.h>
+#include <scheduler/registry.h>
+
+namespace nvfuser {
+
+class Fusion;
+class SchedulerRuntimeInfo;
+class HeuristicSummary;
+
+//! ExprEval scheduler represents the case where we allocate outputs directly using EE. No code is generated.
```

Comment:
Can we save the new scheduler for a separate PR which we will review as soon as the new node is merged?

Reply:
Instead, can we have a very simple test that creates a `FusionExecutor` with this op and evaluates it with ATen. This will let us test the node independent of scheduler changes.